test/root/RootChainManager.test.js

```diff
@@ -196,6 +200,124 @@ contract('RootChainManager', async(accounts) => {
     })
   })
 
+  describe('Deposit Ether', async() => {
+    const depositAmount = mockValues.amounts[1]
+    const depositForAccount = mockValues.addresses[0]
+    const gasPrice = new BN('30000000000')
+    let contracts
+    let rootChainManager
+    let oldAccountBalance
+    let oldContractBalance
+    let depositTx
+    let lockedLog
+    let stateSyncedlog
+
+    before(async() => {
+      contracts = await deployer.deployInitializedContracts()
+      rootChainManager = contracts.root.rootChainManager
+      oldAccountBalance = new BN(await web3.eth.getBalance(accounts[0]))
+      oldContractBalance = new BN(await web3.eth.getBalance(contracts.root.etherPredicate.address))
+    })
+
+    it('Depositor should have proper balance', () => {
+      depositAmount.should.be.a.bignumber.lessThan(oldAccountBalance)
```

Comment:
Used big number for amount everywhere so that we can perform arithmetic safely. It does seem unnecessary in this case. Will take up optimizations in future PR.

Reply:
It is a check for sufficient balance before we try to deposit. It will save unnecessary debugging in case the account doesn't have ether.